paper/MocoPaperAppendix.tex

```diff
@@ -525,36 +516,27 @@ \subsubsection*{Hermite-Simpson transcription}
 The Hermite-Simpson scheme transcribes the optimal control problem into a nonlinear program by approximating integrals using a Hermite interpolant and Simpson integration rule. As a third-order scheme, Hermite-Simpson transcription exhibits accuracy that improves eight-fold when halving the mesh interval.
 
 We use a similar dimensionless time mesh as for the trapezoidal scheme, with $n$ mesh intervals with durations $h_i$. We also introduce collocation points at the midpoints of the mesh intervals, leading to a total of $2n + 1$ time points at which we discretize the continuous variables:
-
 \hermitesimpsontau
+where $ \bar{\tau}_i $ denote mesh interval midpoints.
 
-where $ \bar{\tau}_i $ denote mesh interval midpoints. For conciseness, we define the following functions:
-
+For conciseness, we define the following functions:
 \hermitesimpsonfuncs
-
-where $\mathrm{hermite}_i()$ represents the Hermite interpolant, which yields the value of the midpoint of a segment in a Hermite spline, and $\mathrm{simpson}_i()$ represents the Simpson integration rule. Again, $F$ is a function for mesh interval $i$, and $\eta$ represents any subset of continuous variables.
+where $\mathrm{hermite}_i()$ represents the Hermite interpolant, which yields the value of the midpoint of a segment in a Hermite spline, and $\mathrm{simpson}_i()$ represents the Simpson integration rule. Again, $F$ is a function for mesh interval $i$, and $\eta$ represents any subset of continuous variables. The symbols $\bar{\eta}_i$ represent continuous variables at the mesh interval midpoints, including state variables since here we are using the separated form of Hermite-Simpson transcription~\cite{Betts:2010}.
```

Comment:
I find the wording of "including state variables since..." to be a little confusing. I like the idea of conveying that these are actually variables in the optimization problem, but that might fit better in the sentence we recently added after the NLP is shown.

I added some more description surrounding `\bar{\eta}`. I prefer to not start a sentence with a symbol.